keps/sig-apps/2255-delete-priority/20210112-delete-priority-annotations.yaml

```diff
@@ -0,0 +1,47 @@
+title: KEP Template
```

Comment:
update title

Reply:
also, don't you want to change the directory name to 225-pod-cost since we changed the name of the annotation?

Reply:
done